packages/blade/src/components/Popover/Popover.stories.tsx

```diff
@@ -0,0 +1,696 @@
+            <PopoverInteractiveWrapper display="inline-block">
+              <InfoIcon color="surface.text.normal.lowContrast" size="large" />
+            </PopoverInteractiveWrapper>
+          </Popover>
+
+          <Popover
+            {...args}
+            titleLeading={<LeadingIcon color="surface.text.normal.lowContrast" size="medium" />}
+          >
+            <PopoverInteractiveWrapper>
+              <Badge variant="information">NEW</Badge>
+            </PopoverInteractiveWrapper>
+          </Popover>
+
+          <Popover
+            {...args}
+            titleLeading={<LeadingIcon color="surface.text.normal.lowContrast" size="medium" />}
+          >
+            <PopoverInteractiveWrapper>
+              <Counter value={20} />
+            </PopoverInteractiveWrapper>
+          </Popover>
+        </Box>
+      </Center>
+    </>
+  );
+};
+PopoverInteractiveWrapperTemplate.storyName = 'PopoverInteractiveWrapper';
+PopoverInteractiveWrapperTemplate.args = {
+  placement: 'top',
+  title: 'Settlement breakup',
+  content: <Content />,
+  titleLeading: 'SettlementsIcon',
+};
+
+const MyCustomTriggerButton = React.forwardRef<
+  HTMLDivElement,
+  { children: string } & PopoverTriggerProps
+>(({ children, onTouchEnd, ...props }, ref) => {
+  if (isReactNative()) {
+    return (
+      // just spread the props
+      <Button ref={ref as never} onClick={props.onClick as never}>
+        {children}
+      </Button>
+    );
+  }
+
+  return (
+    // just spread the props
+    <BaseBox
+      backgroundColor="surface.background.level2.lowContrast"
+      padding="spacing.5"
+      borderRadius="medium"
+      role="button"
+      tabIndex={0}
+      ref={ref}
+      style={{ cursor: 'pointer' }}
+      {...props}
+    >
+      {children}
+    </BaseBox>
+  );
+});
+
+const CustomTriggerDocs = () => {
+  if (isReactNative()) return null;
+  return (
+    <List>
+      <ListItem>
+        Make sure to expose ref from the custom component via{' '}
+        <ListItemLink href="https://react.dev/reference/react/forwardRef">
+          React.forwardRef
+        </ListItemLink>
+      </ListItem>
+      <ListItem>
+        Make sure that your component can receive focus{' '}
+        <ListItemText as="span" type="placeholder">
+          (eg: have tabIndex:0)
+        </ListItemText>
+      </ListItem>
+      <ListItem>
+        Forward event handlers to the custom trigger{' '}
+        <ListItemText as="span" type="placeholder">
+          (you can import the PopoverTriggerProps type from blade when using TypeScript)
+        </ListItemText>
+        <List>
+          <ListItem>onClick</ListItem>
+          <ListItem>
+            onMouseDown{' '}
+            <ListItemText as="span" type="placeholder">
+              (not needed if your trigger is a button component)
+            </ListItemText>
+          </ListItem>
+          <ListItem>
+            onPointerDown{' '}
+            <ListItemText as="span" type="placeholder">
```

Comment:
can you make the type as either subtle or muted? placeholder looks very light and hard to read. a11y ❌ 😞